docs/sdks/install-sdks.md

```diff
@@ -33,22 +43,34 @@ npm run dist
 </TabItem>
 <TabItem value="py" label="Python">
 
+:::note
+pyC8 requires Python 3.5+. Python 3.6 or higher is recommended.
+:::
+
+You have several options when installing this SDK.
+
+On macOS, you can use pip3:
+
 ```py
-pyC8 requires Python 3.5+. Python 3.6 or higher is recommended
```

Comment:
For Mac Os is the same command as windows `pip install pyC8` maybe we can merge Mac and Window lines together